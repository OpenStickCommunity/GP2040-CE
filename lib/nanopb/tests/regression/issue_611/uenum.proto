syntax = "proto2";

enum my_enum {
  E0 = 0;
  E240 = 240;
  E1 = -1;
}

enum my_uenum {
  U0 = 0;
  U240 = 240;
}

message my_message_regular {
   required my_enum MyEnum = 1;
   required my_uenum MyUEnum = 2;
}

message my_message_oneof {
  required uint32 Data1 = 1;
  oneof Data {
    bool null = 2;
    my_enum MyEnum = 3;
    my_uenum MyUEnum = 4;
  }
}


