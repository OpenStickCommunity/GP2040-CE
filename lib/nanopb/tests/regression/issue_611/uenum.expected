STATIC.* ONEOF.* ENUM.*MyEnum
STATIC.* ONEOF.* UENUM.*MyUEnum
! [^U]ENUM.*MyUEnum
! UENUM.*MyEnum

